### DISCUSSION TITLE
Plotting differentially abundant taxa over continuous data

### DISCUSSION THREAD

**Discussion by NThie** [FrederickHuangLin/ANCOMBC#85] [Q&A]
Hello! What would be the best way to plot the differentially abundant taxa that ANCOM-BC selected over continuous data? 
Currently, I calculated the log fold change and plotted these as bar plots (like this: https://rdrr.io/bioc/ANCOMBC/f/vignettes/ANCOMBC.Rmd), with positive logfold indicating the abundance of the specific taxa goes up with higher values of my continuous data (positive relationship) and vice versa. But this might be confusing or difficult to interpret because it kind of creates two groups (increase/decrease) - maybe regression/correlation plots would be more informative?

**Comment by Cybernewbye** [comment]
another question on the same topic: as the example on vignette reports age as continuous varables, and lists beta coefficients, I noted that all exp(beta) values, corrected by sign, are within [-1,1] range, and I have the same results on my data.
In the case of continuous variable, is there a relationship between logfold change and correlation coefficients?

**Comment by FrederickHuangLin** [comment]
Hi @NThie,

I think it is a great question and definitely open for discussion. The regression plot is applicable by plotting the residuals (bias-corrected abundances - the effects of other covariates) vs. the continuous covariate.

Hi @Cybernewbye,

The correlation coefficients and the log fold changes (obtained from the regression coefficients) are not the same but share some similarities. You can check this [post](https://stats.stackexchange.com/questions/32464/how-does-the-correlation-coefficient-differ-from-regression-slope) for more details.

Best,
Huang
